cmd/clusterctl/pkg/client/cluster/objectgraph.go

```diff
@@ -231,14 +231,14 @@ func (o *objectGraph) Discovery(namespace string, types []metav1.TypeMeta) error
 			return errors.Wrapf(err, "failed to list %q resources", objList.GroupVersionKind())
 		}
 
-		o.log.V(2).Info(typeMeta.Kind, "Count", len(objList.Items))
+		log.V(5).Info(typeMeta.Kind, "Count", len(objList.Items))
```

Comment:
Does this log need a `msg` as the first argument?

Reply:
No this is intentional. It reads:
Cluster Count=5
Machines Count=15
etc.etc.

Reply:
Oh...you are correct. I'm not sure what I was thinking.